reducer-todo/src/App.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState, useReducer } from 'react';
+import { initialState, todoReducer } from './reducers/Reducer'
+import TodoForm from './components/TodoComponents/TodoForm';
+import TodoList from './components/TodoComponents/TodoList';
+import './components/TodoComponents/Todo.css';
+
+
+const  App = (props) => {
```

Comment:
```suggestion
const  App = () => {
```
You're not using props at this level for anything, so you can take it out.